docs/tutorialkit.dev/src/content/docs/reference/theming.mdx

```diff
@@ -0,0 +1,406 @@
+---
+title: Theming
+description: 'How to theme TutorialKit'
+---
+
+This page describes how to customize the appearance of TutorialKit.
+
+## Logo
+
+You can customize the logo in the top left corner of the TutorialKit UI by replacing the `logo.svg` file in the `public` directory of your TutorialKit project. Supported extensions for the logo are `.svg`, `.png`, `.jpg`, and `.jpeg`.
+
+If you want to use a different logo for dark mode, you can add a `logo-dark` file in the `public` directory. This file is optional, and if it's not provided, TutorialKit will use the `logo` file for both light and dark mode.
+
+## Styling
+
+In order to customize the appearance of TutorialKit, you can use CSS variables to change the colors of various elements. To do this, you can create a `theme.css` file at the root of your TutorialKit project and add the CSS variables you want to customize. You need to define the colors for both light and dark mode.
+
+```css
+:root[data-theme='light'] {
+  /* Light mode colors */
+}
+
+:root[data-theme='dark'] {
+  /* Dark mode colors */
+}
+```
+
+Withing these two root selectors, you can define any of the CSS variables mentioned in the following sections.
+
+These tokens can be used to style the colors of the tutorial on a high level. For more detailed customization, you can use the tokens for the individual elements.
+
+| Token                                   | Description                                  |
+| --------------------------------------- | -------------------------------------------- |
+| `--tk-elements-app-textColor`           | The text color of the TutorialKit app.       |
+| `--tk-elements-app-backgroundColor`     | The background color of the TutorialKit app. |
+| `--tk-elements-app-borderColor`         | The border color of the TutorialKit app.     |
+| `--tk-elements-app-linkColor`           | The link color of the TutorialKit app.       |
+
```

Comment:
Nit: Remove extra newline here.

Reply:
It's personal preference, and also something that is done often in README files, to add 2 newlines in between the main headers to separate it visually while working on the content. But I'll remove them.

Reply:
Oh no, no strong preference. I am personally just not a big fan of extra newlines but if we want them to more visually separate them then let's keep them. It's fine but we should be consistent then that's all 👍

Reply:
I'll check the other documents. If we have it, then I'll leave it in, otherwise I'll remove them.